pkg/action/scan.go

```diff
@@ -36,7 +36,12 @@ func findFilesRecursively(ctx context.Context, root string) ([]string, error) {
 				clog.FromContext(ctx).Errorf("walk %s: %v", path, err)
 				return err
 			}
-			if info.IsDir() || strings.Contains(path, "/.git/") {
+			if info.IsDir() ||
+				// Ignore the source tree and other bincapz scan artifacts
+				strings.Contains(path, "/.git/") ||
+				strings.Contains(path, ".mdiff") ||
```

Comment:
I'd rather not ignore more things unless necessary - in fact, I kind of regret ignoring /.git/. Is there any way around doing this?

The exception wasn't for bincapz, it was for scanning git repositories of malware.

Reply:
Definitely. There's no reason we can't scan our scan artifacts.

Reply:
Reverted in [`7d5d2a1` (#380)](https://github.com/chainguard-dev/bincapz/pull/380/commits/7d5d2a1e8efa84955aa20ce263400bd6be70c659).